ConsistencyManagerTests/MemoryWarningTests.swift

```diff
@@ -42,5 +54,10 @@ class MemoryWarningTests: ConsistencyManagerTestCase {
             XCTAssertEqual(listenersArray.count, 0)
         }
         // Else it's nil which is fine too
+
+        // Now, let's check we got the right start/finish times
+        XCTAssertEqual(cleanMemoryStartedTimes.count, 1)
+        XCTAssertEqual(cleanMemoryFinishedTimes.count, 1)
+        XCTAssertTrue(cleanMemoryStartedTimes[0].timeIntervalSince1970 <= cleanMemoryFinishedTimes[0].timeIntervalSince1970)
```

Comment:
One more test case to add to make sure the notifications fire at expected times, register test start time, and at the end assert that cleanMemoryStartedTimes happened after that time.